common/djangoapps/student/models.py

```diff
@@ -1603,6 +1605,19 @@ def emit_event(self, event_name, enterprise_uuid=None):
                 if is_personalized_recommendation is not None:
                     segment_properties['is_personalized_recommendation'] = is_personalized_recommendation
 
+                # TODO: VAN-1052 - This is Optimizely's A/B experimentation block to test welcome email redesign.
+                #  Remove this temporary block after pausing the experiment.
+                optimizely_experiment_variation = None
+                if optimizely_client:
+                    optimizely_experiment_variation = optimizely_client.activate(
+                        'welcome_email_redesign_experiment', str(self.user.id)
```

Comment:
We should use global variable for `welcome_email_redesign_experiment ` because this would be used on other places as well.
i.e `OPTIMIZELY_EXPERIMENT_NAME = 'welcome_email_redesign_experiment'`